pkg/apiserver/registry/groupmember/rest.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package groupmember
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/registry/rest"
+
+	"antrea.io/antrea/pkg/apis/controlplane"
+	"antrea.io/antrea/pkg/apiserver/registry/networkpolicy"
+)
+
+type REST struct {
+	querier groupMembershipQuerier
+}
+
+var (
+	_ rest.Storage           = &REST{}
+	_ rest.Scoper            = &REST{}
+	_ rest.GetterWithOptions = &REST{}
+)
+
+// NewREST returns a REST object that will work against API services.
+func NewREST(querier groupMembershipQuerier) *REST {
+	return &REST{querier}
+}
+
+type groupMembershipQuerier interface {
+	GetGroupMembersByNamespace(name, namespace string) (controlplane.GroupMemberSet, []controlplane.IPBlock, error)
```

Comment:
I feel the "ByNamespace" is a bit misleading. Although making it more accurate like ByNamespacedName could fix it, I wonder if we could just merge the two arguments and just use `GetGroupMembers` given the callee doesn't need to separate it anyway, we could comment that the method accepts both ClusterGroup name and Group name.

Or we could rename existing GetGroupMembers to GetClusterGroupMembers which takes ClusterGroup name and add GetGroupMembers which takes Group Namespace and Name.